tests/fixtures/descriptor_tests/with_create_descriptor_hook/core/core_api.yml

```diff
@@ -0,0 +1,5 @@
+location:
```

Comment:
This is the setup for the `bootstrap.py` test. We're setting up bundles with versions and/or names that do not exist in the app store. If our hook failed to do its job or wasn't invoked, the test would fail due to the bundles not being found on the app store.